spec/components/find/courses/entry_requirements_component/view_spec.rb

```diff
@@ -3,6 +3,121 @@
 require 'rails_helper'
 
 describe Find::Courses::EntryRequirementsComponent::View, type: :component do
+  let(:course) { build(:course, subjects:) }
+  let(:subjects) { [build(:secondary_subject, subject_name)] }
+  let(:result) { render_inline(described_class.new(course: course.decorate)) }
+
+  context 'when english is selected' do
+    let(:subject_name) { :english }
+
+    it 'renders correct message' do
+      expect(result.text).to include('or you’ve not used your subject knowledge in a while, you may be asked to complete a')
```

Comment:
we could dry up some of these values which are being asserted against. Eg The `or you’ve not used your subject knowledge in a while, you may be asked to complete a'` is the same for most of the scenarios so it could go in an `let(:ske_text) { 'or you’ve not used your subject knowledge in a while, you may be asked to complete a' }` which could then be re-used.

Reply:
I've made these changes